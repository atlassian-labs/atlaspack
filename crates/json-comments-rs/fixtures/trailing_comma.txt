{
  "a1": [1,],
  "a2": [1,/* x */],
  "a3": [
      1, // x
  ],
  "o1": {v:1,},
  "o2": {v:1,/* x */},
  "o3": {
      "v":1, // x
  },
  # another
}
