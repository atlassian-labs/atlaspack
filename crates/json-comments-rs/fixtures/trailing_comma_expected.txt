{
  "a1": [1 ],
  "a2": [1        ],
  "a3": [
      1      
  ],
  "o1": {v:1 },
  "o2": {v:1        },
  "o3": {
      "v":1      
  } 
           
}
